crates/telio-crypto/src/lib.rs

```diff
@@ -60,10 +97,8 @@ pub fn smaller_key_in_meshnet_canonical_order<'a>(
 }
 
 /// Preshared key type
-#[derive(
-    Default, PartialOrd, Ord, PartialEq, Eq, Hash, Copy, Clone, DeserializeFromStr, SerializeDisplay,
-)]
-pub struct PresharedKey(pub [u8; KEY_SIZE]);
+#[derive(Default, Debug, PartialOrd, Ord, PartialEq, Eq, Hash, Copy, Clone, DeserializeFromStr)]
```

Comment:
Why is `SecretKey` deriving `Deserialize, Serialize` but the preshared key derives `DeserializeFromStr`? Do they have different deser requirements?

Reply:
`DeserializeFromStr` wasn't needed - I've removed it.